build.gradle.kts

```diff
@@ -32,6 +32,7 @@ dependencies {
     implementation(project(":better_direnv-products-idea"))
     implementation(project(":better_direnv-products-nodejs"))
     implementation(project(":better_direnv-products-shellscript"))
+    implementation(project(":better_direnv-products-pycharm"))
```

Comment:
Nitpick: Rename this Python, since it should be universally supported across any IDEs with the Python plugin installed.

Reply:
You got it - I wasn't sure because the Go version was named goland. Would you like me to rename the run config extension as well?

Reply:
Good call, I should probably rename that. Yeah, if you can rename that as well that'd be great.